pkg/constant/constants.go

```diff
@@ -19,4 +19,12 @@ const (
 	Referer                       = "Referer"
 	UserAgent                     = "User-Agent"
 	EnvSubdomainValidationEnabled = "SUBDOMAIN_VALIDATION_ENABLED"
+	DisableDevstackTracerEnv      = "prod"
```

Comment:
DisabledDevstackTracerEnv

Reply:
1. the hardcoded prod is not fit. 
2. btw, DisableDevstackTracerEnv, why the stack, s is not capital

Reply:
@arya-ab In Java we have `REALM_LIVE` to determine of which env is included as Live since we have prod,live,demo etc
Might be good to standardize the same. In Java the value is separated by comma.

Reply:
Could you help point me to the code in java repo @rexydwybrundy 

anyway, I just thought. In Live env, does the REALM_NAME == REALM_LIVE?
if that the case, we can make it as constrain I think

Reply:
REALM_NAME is defined by the deployment. It determine which realm the service is deployed too
REALM_LIVE indicate which env considered as Live environment
The checking is something like here
https://bitbucket.org/accelbyte/justice-server-base/commits/65c99ff2256db0cb767bc801085f2f448435fc3a#chg-shared-dependencies.gradle